docker-toolchain/Dockerfile

```diff
@@ -6,11 +6,12 @@ LABEL maintainer='Charlton Rodda'
 ARG UNAME=builder
 ARG UID=1000
 ARG GID=1000
+ARG ostype=Linux
 
 RUN apt-get -qq update
 RUN apt-get -qq install curl build-essential gcc-arm-linux-gnueabihf vim
 
-RUN groupadd -g $GID $UNAME
+RUN bash -c 'if [ ${OSTYPE} == Linux ]; then groupadd -r --gid ${GID} ${UNAME}; fi'
```

Comment:
Environment variables are case sensitive, and I assume these are too. Above `ostype` is set, not `OSTYPE`. They should be made consistent.